index.js

```diff
@@ -99,6 +101,12 @@ bot.Dispatcher.on(Events.GATEWAY_READY, () => {
   setInterval(() => {
     bot.Users.fetchMembers() // Hacky way to cache offline users, #blamelazyloading
   }, 600000)
+
+  // admin-queue amount playing status
+  setInterval(() => {
+    r.db('DFB').table('queue').distinct().count().run().then(c => bot.User.setGame(c.toString() + " reports in queue"))
```

Comment:
Shouldn't this be filtered to only `adminMergeRequest` and `adminReviewDelete`? Because aren't [newCard](https://github.com/SteamingMutt/DiscordFeedback/blob/master/Commands/Commands/queue.js#L16) and [chatVote](https://github.com/SteamingMutt/DiscordFeedback/blob/master/Commands/Commands/queue.js#L40) also in the queue table?

If so then use something along the lines of: `.filter(r.row('type').eq('adminMergeRequest').or(r.row('type').eq('adminReviewDelete')))`

Correct me if I'm wrong, please @SteamingMutt

Reply:
That's right, but that's the wrong solution to filtering it, I'd actually want to put in

.without('newCard').without('chatVote')

I'm away from my PC till morning (7:00-9:00AM EST) so I won't be able to fix it for the moment.

Reply:
Ah, didn't know that was a thing, so:

`.without({ type: 'newCard' }).without({ type: 'chatVote' })`

Reply:
That type bit is unneeded, you just need to pass the table name string, and that'll be enough.